.github/workflows/CI.yml

```diff
@@ -37,17 +37,20 @@ jobs:
           - 'nightly'
         group: [ 'short', 'long' ]
         os:
-          - ubuntu-latest
+          - [Linux, RPTU]
```

Comment:
Can we do that (I am guessing not, but perhaps worth a try?)
```suggestion
          - [Linux, RPTU, ${{ matrix.group }}]
```

Reply:
While this may not work, note that we use the `os` field for just one thing (I believe), namely in line 28: `runs-on: ${{ matrix.os }}`.

So in theory it should be possible to use a more complex expression there which takes both `matrix.os` and `matrix.group` into account. You may have to rethink what kind of string is put into `matrix.os`, though.

Reply:
Perhaps change the `matrix.os` values to just plain `Linux` or `macOS` and then change `runs-on: ${{ matrix.os }}` to
```
runs-on: [ ${{ matrix.os }}, RPTU, ${{ matrix.group }}$ ]
```

Reply:
(OK, to be fair, the docs do suggest that it may be possible to write `${{ matrix.os[42] }}` to access the object at position 41 (0-indexed? 1-indexed?) of `matrix.os` if it is a list. But still no idea how to find out whether it is a list (which we could avoid if we just make sure it is always one) nor how to append to it.

Reply:
OK, I  had to search for "array" and it does mention a bit more: we can also use `contains` to check if an array contains some element; and `join` to turn an array into a list. Then there is also `fromJSON` and `format`.... So one could do this evil thing: take the list, turn it into a string via `join` using commas as seperators; use format to "append" to this flattened list, and put `[` and `]` around it. This now is valid JSON and we can use `fromJSON` to turn it back into the desired list with one element added.

Of course we will also be top on the list of people to be exterminated when the machines rise.